runtime/lua/vim/lsp.lua

```diff
@@ -4,15 +4,15 @@ local lsp_rpc = require('vim.lsp.rpc')
 local protocol = require('vim.lsp.protocol')
 local util = require('vim.lsp.util')
 local sync = require('vim.lsp.sync')
-local api = vim.api
 
 local vim = vim
+local a = vim.api
```

Comment:
I would prefer `api` over `a`; the latter makes the code a bit harder to read. 

Matter of taste, though, and I'd be interested in hearing other opinions from @neovim/core.